### DISCUSSION TITLE
cannot write to files in a micro

### DISCUSSION THREAD

**Discussion by akhilyannabathula** [deta/cloud-docs#556] [Help]
Hi, 
I'm trying to append data to a file using python end point.
It was throwing an error "Read only file", tried changing access of the file but unable to do so as the folder is also a read only folder.
Appreciate your help.

error - "Read-only file system: './clothe_store.db'"

**Comment by jnsougata** [comment]
> Hi, 
> 
> I'm trying to append data to a file using python end point.
> 
> It was throwing an error "Read only file", tried changing access of the file but unable to do so as the folder is also a read only folder.
> 
> Appreciate your help.
> 
> 
> 
> error - "Read-only file system: './clothe_store.db'"

Micro runtime is read-only. use deta base or drive as database
